repos/spack_repo/builtin/packages/kokkos_fft/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack.package import *
+
+
+class KokkosFft(CMakePackage):
+    """FFT interfaces for Kokkos C++ Performance Portability Programming EcoSystem"""
+
+    homepage = "https://github.com/kokkos/kokkos-fft"
+    url = "https://github.com/kokkos/kokkos-fft/archive/refs/tags/v0.3.0.tar.gz"
+
+    maintainers("cedricchevalier19", "tpadioleau", "yasahi-hpc")
+
+    license("Apache-2.0 WITH LLVM-exception OR MIT", checked_by="cedricchevalier19")
+
+    version("0.3.0", sha256="a13c423775afec5f9f79fa9a23dd6001d3d63bae9f4786b1e0cd3ed65b3993a3")
+
+    variant("host_backend", default="fftw-serial",
+            values=("fftw-serial", "fftw-openmp"),
+            multi=True,
+            description="Enable host backend")
+    variant("device_backend", default="none",
+            values=("none", "cufft", "hipfft", "onemkl"),
+            multi=False,
+            description="Enable device backend")
+    variant("tests", default=False, description="Enable tests")
+
+    depends_on("kokkos@4.4:4 +complex_align")
```

Comment:
You should also add a dependency to `cxx`
```suggestion
    depends_on("cxx", type="build")

    depends_on("kokkos@4.4:4 +complex_align")
```

Reply:
yes very true

Reply:
done in 285e88b4d3b288d4f4b9800d2808ed1aaabe5413